src/System.Diagnostics.Process/src/System/Diagnostics/ModuleInfo.cs

```diff
@@ -12,10 +12,12 @@ namespace System.Diagnostics
     /// <internalonly/>
     internal sealed class ModuleInfo
     {
+#pragma warning disable 0649 // unused fields on some platforms
```

Comment:
I've done it this way because I think some of this could potentially be supported at some point, I'm just not sure how to do it at present.